pyproject.toml

```diff
@@ -37,10 +37,11 @@ toml = "^0.10.1"
 srvresolver = "^0.3.5"
 colorama = "^0.4.4"
 
+goth = {git = "https://github.com/golemfactory/goth.git", branch = "master", optional = true, python = "^3.8.0"}
```

Comment:
I think that there should be a comment explaining why `goth` was added as an optional dependency, not a dev dependency.

@azawlocki:
> I couldn't make it work with goth as dev-dependency, as goth requires Python >= 3.8, which conflicts with Python >= 3.6 for the whole project. So goth is now installed with optional = "true" and python = "^3.8.0".
> Perhaps it can be moved to dev-dependencies (it would be an optional dev-dependency then, I'm not sure if that would work though).

Reply:
a comment added in https://github.com/golemfactory/yapapi/pull/281/commits/0d80b77dcf376c1adc98da244f60945d4daf27c7